content/_layouts/developerguide.html.haml

```diff
@@ -3,7 +3,7 @@ layout: developerbook
 ---
 
 %p
-  %a{:href => '../../guides'}
+  %a{:href => '/doc/developer/guides'}
```

Comment:
Not sure about that one, I thought we wanted to support running the site on a prefix?

Reply:
As far as I can tell, we have several haml files and several adoc files which href to /doc/developer .  I don't think this is enough to block the merge.

Reply:
Definitely was not intended a blocker.

Reply:
Thanks for checking! With the last commit it's changed to `expand_link(...)`. Note that normal relative link won't do because the side navigation is used on different levels:
https://jenkins.io/doc/developer/book/
https://jenkins.io/doc/developer/publishing/style-guides/

Reply:
I think use from `/doc/developer/book` may be a bug out of laziness, I called it `developerguide` for a reason.